### PR TITLE
added python-keycloak recipe

### PR DESCRIPTION
**Pull request by edennuriel**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/python-keycloak/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  requires:
+    - httmock >=1.2.5
+
+about:
+  home: "https://github.com/marcospereirampj/python-keycloak"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "python-keycloak is a Python package providing access to the Keycloak API."
+  doc_url:
```
Comment (hmaarrfk) [inline recipes/python-keycloak/meta.yaml]: these two lines can be removed.

File: recipes/python-keycloak/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - python-jose >=1.4.0
```
Comment (hmaarrfk) [inline recipes/python-keycloak/meta.yaml]: Can you double check if python-jose and requests are needed during the installation phase?

File: recipes/python-keycloak/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: c1ffcdfd8f770774c02b7cbfacdba9d7ef94c0be4e42fabe49308e436ba21490
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (hmaarrfk) [inline recipes/python-keycloak/meta.yaml]: This seems to require python 3

File: recipes/python-keycloak/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c1ffcdfd8f770774c02b7cbfacdba9d7ef94c0be4e42fabe49308e436ba21490
+
+build:
+  number: 0
```
Comment (hmaarrfk) [inline recipes/python-keycloak/meta.yaml]: It also seems that this package might be a "noarch" package
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

File: recipes/python-keycloak/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - keycloak.authorization
+    - keycloak.tests
+  requires:
+    - httmock >=1.2.5
+
+about:
+  home: "https://github.com/marcospereirampj/python-keycloak"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (hmaarrfk) [inline recipes/python-keycloak/meta.yaml]: Please open an issue with upstream asking them to create a Manifest.in file with
```
include LICENSE.txt
```
in it.

**Comment by synapticarbors** [comment]
I've added a PR upstream to package the license file: 

https://github.com/marcospereirampj/python-keycloak/pull/68

File: recipes/python-keycloak/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+about:
+  home: "https://github.com/marcospereirampj/python-keycloak"
+  license: MIT
+  license_family: MIT
+  # PR upstream to package license https://github.com/marcospereirampj/python-keycloak/pull/68
+  license_file: LICENSE.txt
+  summary: "python-keycloak is a Python package providing access to the Keycloak API."
+  doc_url: 
+  dev_url:
```
Comment (scopatz) [inline recipes/python-keycloak/meta.yaml]: Please remove empty entries

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**